### ISSUE TITLE
Move all bot developer only commands.

### ISSUE THREAD

**Issue by THEGOLDENPRO** [Goldy-Bot/Goldy-Bot-V5#81]
As the title says, I would like to move all Dev only commands like /extensions to a one admin command that can only be used by the bot Dev and registered admins in DMs.

**Comment by THEGOLDENPRO** [comment]
keyword: **DMs**

**Comment by THEGOLDENPRO** [comment]
Waiting on #88

**Comment by THEGOLDENPRO** [comment]
closing as this issue is no longer needed
